lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -3809,6 +3826,7 @@ void ByteCodeGenerator::StartEmitFunction(ParseNode *pnodeFnc)
                 !funcInfo->Escapes() &&
                 funcInfo->frameObjRegister != Js::Constants::NoRegister &&
                 !ApplyEnclosesArgs(pnodeFnc, this) &&
+                funcInfo->paramScope->GetCanMergeWithBodyScope() && // There is eval in the param scope
```

Comment:
Do we want to explicitly check that this case triggers because there is an eval?